lib/app.rb

```diff
@@ -53,6 +53,22 @@ def exit
     puts 'Thanks for using this app.'
   end
 
+  def list_genres
+    @items[:music_album] = read_music_album(@items[:music_album])
```

Comment:
### Recommendation
I believe you don't need the parameter for the method #read_music_album. If you can remove it it will be easier to understand. If it can't be removed it also works.

Reply:
Hello @Vgonma I also think now I don't need to send a parameter but when I quit I get this error (undefined method `empty?' for nil:NilClass) coming from line preserve_music_album.rb:24. 
But as I told you I also think I don't need to pass the data maybe I need to refactor my code but for now I am going to keep in this way. 
Thanks for the feedback